[12.0][ADD] New delivery note modules (#1840)
OCA/l10n-italy: l10n_it_delivery_note_order_link/readme/DESCRIPTION.rst

**English**

> This module manage the Italian DDT.
> 
> Link from purchase orders linked delivery notes

Reviewer: ```suggestion
This module link delivery notes to purchase orders
```

**Italiano**

> Questo modulo consente di gestire i DDT.

Reviewer: ho lasciato un suggerimento per togliere il refuso

> Questo modulo gestisce il DDT italiano.
> 
> Consente di collegare gli ordini di acquisto ai DDT.

Reviewer: ```suggestion
Questo modulo consente di collegare gli ordini di acquisto ai DDT.
```
